azuredevops/provider.go

```diff
@@ -23,48 +23,49 @@ import (
-			"azuredevops_build_permissions":                 permissions.ResourcePipelinePermissions(),
-			"azuredevops_serviceendpoint_githubapp":         githubapp.ResourceGithubApp(),
+			"azuredevops_resource_authorization":               build.ResourceResourceAuthorization(),
+			"azuredevops_branch_policy_build_validation":       policy.ResourceBranchPolicyBuildValidation(),
+			"azuredevops_branch_policy_min_reviewers":          policy.ResourceBranchPolicyMinReviewers(),
+			"azuredevops_branch_policy_auto_reviewers":         policy.ResourceBranchPolicyAutoReviewers(),
+			"azuredevops_branch_policy_work_item_linking":      policy.ResourceBranchPolicyWorkItemLinking(),
+			"azuredevops_branch_policy_comment_resolution":     policy.ResourceBranchPolicyCommentResolution(),
+			"azuredevops_branch_policy_merge_types":            policy.ResourceBranchPolicyMergeTypes(),
+			"azuredevops_build_definition":                     build.ResourceBuildDefinition(),
+			"azuredevops_project":                              core.ResourceProject(),
+			"azuredevops_project_features":                     core.ResourceProjectFeatures(),
+			"azuredevops_variable_group":                       taskagent.ResourceVariableGroup(),
+			"azuredevops_serviceendpoint_artifactory":          serviceendpoint.ResourceServiceEndpointArtifactory(),
+			"azuredevops_serviceendpoint_aws":                  serviceendpoint.ResourceServiceEndpointAws(),
+			"azuredevops_serviceendpoint_azurerm":              serviceendpoint.ResourceServiceEndpointAzureRM(),
+			"azuredevops_serviceendpoint_bitbucket":            serviceendpoint.ResourceServiceEndpointBitBucket(),
+			"azuredevops_serviceendpoint_azuredevops":          serviceendpoint.ResourceServiceEndpointAzureDevOps(),
+			"azuredevops_serviceendpoint_dockerregistry":       serviceendpoint.ResourceServiceEndpointDockerRegistry(),
+			"azuredevops_serviceendpoint_azurecr":              serviceendpoint.ResourceServiceEndpointAzureCR(),
+			"azuredevops_serviceendpoint_github":               serviceendpoint.ResourceServiceEndpointGitHub(),
+			"azuredevops_serviceendpoint_github_enterprise":    serviceendpoint.ResourceServiceEndpointGitHubEnterprise(),
+			"azuredevops_serviceendpoint_kubernetes":           serviceendpoint.ResourceServiceEndpointKubernetes(),
+			"azuredevops_serviceendpoint_runpipeline":          serviceendpoint.ResourceServiceEndpointRunPipeline(),
+			"azuredevops_serviceendpoint_sonarqube":            serviceendpoint.ResourceServiceEndpointSonarQube(),
+			"azuredevops_serviceendpoint_ssh":                  serviceendpoint.ResourceServiceEndpointSSH(),
+			"azuredevops_serviceendpoint_npm":                  serviceendpoint.ResourceServiceEndpointNpm(),
+			"azuredevops_serviceendpoint_genericwebhook":       serviceendpoint.ResourceServiceEndpointGenericWebhook(),
+			"azuredevops_serviceendpoint_babylonawsassumerole": serviceendpoint.ResourceServiceEndpointBabylonAWSAssumeRole(),
```

Comment:
https://github.com/babylonhealth/azure-extensions/blob/master/vss-extension.json#L54

Reply:
CI seems to be running an fmt check: https://github.com/babylonhealth/terraform-provider-azuredevops/blob/master/GNUmakefile#L28